#! /bin/sh /usr/share/dpatch/dpatch-run
## link_system_libs.dpatch by Mirco Bauer <meebey@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Link against system libraries in preference to versions bundled in the
## DP: f-spot tarball.

Index: f-spot.git/Makefile.include
===================================================================
--- f-spot.git.orig/Makefile.include	2009-10-29 14:18:29.542756809 +0000
+++ f-spot.git/Makefile.include	2009-10-29 14:19:05.509756682 +0000
@@ -3,9 +3,6 @@
 
 ## Directories
 
-DIR_ADDINS_ADDINS = $(top_builddir)/lib/mono-addins/Mono.Addins
-DIR_ADDINS_GUI = $(top_builddir)/lib/mono-addins/Mono.Addins.Gui
-DIR_ADDINS_SETUP = $(top_builddir)/lib/mono-addins/Mono.Addins.Setup
 DIR_DOCS = $(top_builddir)/docs
 DIR_EXTENSIONS = $(top_builddir)/extensions
 DIR_GLITZ = $(top_builddir)/lib/glitz-sharp/src
@@ -24,10 +21,10 @@
 LINK_KEYRING = -r:$(DIR_KEYRING)/gnome-keyring-sharp.dll
 LINK_GLITZ = -r:$(DIR_GLITZ)/NDesk.Glitz.dll 
 LINK_GPHOTO2 = -r:$(DIR_GPHOTO2)/libgphoto2-sharp.dll
-LINK_MONO_ADDINS =					\
-	-r:$(DIR_ADDINS_ADDINS)/Mono.Addins.dll		\
-	-r:$(DIR_ADDINS_SETUP)/Mono.Addins.Setup.dll	\
-	-r:$(DIR_ADDINS_GUI)/Mono.Addins.Gui.dll
+LINK_MONO_ADDINS =			\
+	-pkg:mono-addins		\
+	-pkg:mono-addins-setup		\
+	-pkg:mono-addins-gui
 LINK_SEMWEB = -r:$(DIR_SEMWEB)/SemWeb.dll
 LINK_GIOSHARP = -r:$(DIR_GIOSHARP)/gio-sharp.dll
 LINK_GTKSHARPBEANS = -r:$(DIR_GTKSHARPBEANS)/gtk-sharp-beans.dll
Index: f-spot.git/configure.ac
===================================================================
--- f-spot.git.orig/configure.ac	2009-10-29 14:18:29.578756336 +0000
+++ f-spot.git/configure.ac	2009-10-29 14:19:05.509756682 +0000
@@ -308,10 +308,6 @@
 lib/libgphoto2-sharp/Makefile
 lib/libjpegtran/Makefile
 lib/Makefile
-lib/mono-addins/Makefile
-lib/mono-addins/Mono.Addins.Gui/Makefile
-lib/mono-addins/Mono.Addins/Makefile
-lib/mono-addins/Mono.Addins.Setup/Makefile
 lib/semweb/Makefile
 docs/Makefile
 icons/Makefile
Index: f-spot.git/lib/Makefile.am
===================================================================
--- f-spot.git.orig/lib/Makefile.am	2009-10-29 14:18:29.562758733 +0000
+++ f-spot.git/lib/Makefile.am	2009-10-29 14:19:05.509756682 +0000
@@ -10,7 +10,6 @@
 	libjpegtran			\
 	libfspot			\
 	libgphoto2-sharp	\
-	mono-addins			\
 	semweb
 
 # Depending on whether the packaged ndesk-dbus is used, we might not want to
